trunk/source/CACSL2BoogieTranslator/src/de/uni_freiburg/informatik/ultimate/cdt/translation/implementation/base/CHandler.java

```diff
@@ -710,13 +710,7 @@ public Result visit(final IDispatcher main, final IASTArraySubscriptExpression n
 	}
 
 	public Result visit(final IDispatcher main, final IASTASMDeclaration node) {
-		if (mSettings.isSvcompMode()) {
-			// workaround for now: ignore inline assembler instructions
-			return new SkipResult();
-		}
-		final String msg = "CHandler: Not yet implemented: \"" + node.getRawSignature() + "\" (Type: "
-				+ node.getClass().getName() + ")";
-		throw new UnsupportedSyntaxException(mLocationFactory.createCLocation(node), msg);
+		return new SkipResult();

```

Comment:
we now always ignore assembler without any warning? 
at least add a log message

Reply:
The same happened before in SV-COMP mode. So we should probably not throw an exception here. But always logging the previous error message seems like a good idea!

Reply:
The solution would be a boolean setting `Ignore inline assembler`, right?

Reply:
Yes, I think those are the two options: Always succeed with a warning or add a setting to crash. I am fine with both

Reply:
We should not add another setting for that. I would propose we generate a result that warns a user that we encountered such an instruction and ignored it (i.e., the result became potentially unsound). Same for preprocessor pragmas.

Reply:
Okay, `CTranslationResultReporter.warn` does exactly that, but only if the setting `Report unsoundness warnings` is enabled, per default it is however disabled. Should we just change the default to enabled?

Reply:
Probably a good idea. Shouldnt be that many, right?

Reply:
I don't think so. And in all cases, where this is used, it should be helpful to report the unsoundness (with a logger-warning and a result).